api/src/main/proto/spine/protodata/events.proto

```diff
@@ -264,3 +265,15 @@ message RpcExited {
 
     RpcName rpc = 3;
 }
+
+// Emitted when a Protobuf dependency file is discovered.
+//
+// Normally, no code should be generated for dependencies. However, they can be used for additional
+// info when generating code for the module's own types.
+//
+message DependencyDiscovered {
```

Comment:
I would be more precise with this name, so that Java- and Proto-level dependencies could be distinguished.

Reply:
I've updated the doc somewhat. I'd like to avoid adding the `Proto` prefix because: 
 - we only work with Protobuf, no Java or JS or other dependency is ever considered in ProtoData (not talking about the Gradle compiler here, only ProtoData proper);
 - this is one of a series of events such as `TypeEntered`, `OptionDiscovered`, etc., which do not specify the language eiher.